src/erlcloud_ec2.erl

```diff
@@ -1188,6 +1188,7 @@ extract_image(Node) ->
      {description, get_text("description", Node)},
      {root_device_type, get_text("rootDeviceType", Node)},
      {root_device_name, get_text("rootDeviceName", Node)},
+     {creation_date, erlcloud_xml:get_time("creationDate", Node)},
```

Comment:
I had previously checked for an existing describe_instances test. There does not appear to be one.  I can probably add one, but I'm very new to erlang, so it will be a bit of a job for me. I did test the make sure it works for my code that is using it. For an example, would you like to me to add a whole image term as a comment like some of the other functions have? An image term is pretty lengthy.